src/third_party/progress/progress.hpp

```diff
@@ -208,6 +216,11 @@ class C_Progress_display : public C_Progress
     } // display_tic
 };
 
+inline C_Progress& C_Progress::dummy()
+{
+  static C_Progress null;
```

Comment:
Nice! I wasn't aware of std::call_once. I will have to add a .cpp file, right? Otherwise flag will have multiple definitions